e2e/tests/test_e2e.py

```diff
@@ -221,6 +221,8 @@ def test_enable_disable_connection_pooler(self):
         self.eventuallyEqual(lambda: k8s.count_services_with_label(
                             'application=db-connection-pooler,cluster-name=acid-minimal-cluster'),
                             2, "No pooler service found")
+        self.eventuallyEqual(lambda: k8s.count_secrets_with_label('application=spilo,cluster-name=acid-minimal-cluster'),
+                             5, "Secret not created")
```

Comment:
Can you maybe include the `db-connection-pooler` label to only get the secret of interest? Just in case the number of the other secrets changes we don't have to care about the code here.